### PR TITLE
Create popover component

### LINKED ISSUE #70: Implement dropdown component

### PR DESCRIPTION
**Pull request by bramvanhoutte**
Closes #70

### CODE REVIEW

File: src/components/Popover/Popover.test.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    );
+
+    const component = (
+      <Popover isVisible onClose={() => {}} content={content}>
+        <Button>Click me!</Button>
+      </Popover>
+    );
+    const { asFragment, getByTestId } = render(component);
+
+    await waitFor(() => getByTestId('menu-item-default'));
```
Comment (RobinWijnant) [inline src/components/Popover/Popover.test.tsx:23]: Is it possible to write it without `await waitFor()`?

File: src/components/Popover/Popover.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  const divRef = useRef<HTMLDivElement>(null);
+  const menuRef = useRef<HTMLDivElement>(null);
+
+  const [arrowRef, setArrowRef] = useState<HTMLDivElement | null>(null);
+
+  useOutsideClick(() => {
+    onClose();
+  }, menuRef);
+
+  const { styles: popperStyles, attributes } = usePopper(divRef.current, menuRef.current, {
```
Comment (RobinWijnant) [inline src/components/Popover/Popover.tsx:35]: wat een ref sletje die usePopper 😀

File: src/utils/hooks/useOutsideClick.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React, { useEffect } from 'react';
+
+export const useOutsideClick = (callback: Function, elementRef: React.RefObject<any>) => {
```
Comment (RobinWijnant) [inline src/utils/hooks/useOutsideClick.ts:3]: Nice steal 😀. I also used this a week ago at work

**Review by RobinWijnant — APPROVED** [review]

File: src/components/Popover/Popover.test.tsx (hunk 1 above)
Comment (bramvanhoutte) [reply src/components/Popover/Popover.test.tsx:23]: No, it complains about an update that has to be wrapped in "act"

**Pull request merged**